### PR TITLE
Fix for isInteger function #278

### PR DESCRIPTION
**Pull request by vikene**
Fix for issue #278.

Closes #theRelatedIssueNumber

- [] Running `yarn lint` does not trigger any linter errors
- [] The test of the method you have fixed is passing
- [] You have written a new skeleton method for someone else to work on!
- [] You have written tests to accompany your skeleton method

### CODE REVIEW

**Comment by vikene** [comment]
Sounds great, I will do that 👍

File: lib/isInteger.js [hunk 1]
```diff
@@ -3,7 +3,18 @@
 // @returns {Boolean}
 
 function isInteger(param) {
-  return;
+  if(typeof param == "number"){
+    if((param % 1) === 0)
```
Comment (BlakeGuilloud) [inline lib/isInteger.js]: Also- correct me if I'm wrong (it's late for me), but aren't negative numbers integers? This check would return false if `param === -123`.

Comment (BlakeGuilloud) [reply lib/isInteger.js]: Maybe add that to the test suite!

Comment (BlakeGuilloud) [reply lib/isInteger.js]: never mind.. I'm an idiot. `-0 === 0` Goodnight javascript!

**Comment by ktilcu** [comment]
@vikene I would love to get this merged in but I don't see any new methods created for someone else to work on. If you add those then I can get this merged in. Thanks for the work so far!

**Comment by vikene** [comment]
Something got messed with PR on my end. It got created as new PR. Here is the link https://github.com/BlakeGuilloud/ganon/pull/425. Please discard this one. sorry about this.

**Pull request closed without merging**